### PR TITLE
Add ability to open carousel and select an owned spell from empty sheet slot

### PR DESCRIPTION
**Pull request by bqx**

### CODE REVIEW

File: src/components/CarouselClassSpells/index.tsx [hunk 1]
```diff
@@ -15,15 +15,16 @@ export interface Spell {
   description: string;
 }
 
-export default function SpellbookCarousel({ onClose, onSelectItem }: Props) {
+// Display available class spells
+export default function EquipClassSPells({ onClose, onSelectItem }: Props) {
```
Comment (AndaxH) [inline src/components/CarouselClassSpells/index.tsx]: EquipClassSpells *

File: src/components/EmptyCharacterSheetSlot/index.tsx [hunk 2]
```diff
@@ -1,18 +1,43 @@
...
 }
 
 const EmptyCharacterSheetSlot = (props: Props) => {
+  const { powers } = useContext(GameContext);
+  const { value: isModalOpen, toggle: toggleModal } = useFlipFlop(false);
+
+  const handleSelectItem = useCallback(
+    (id: CardId) => {
+      const power = getPowerById(id);
+      console.log(power);
```
Comment (AndaxH) [inline src/components/EmptyCharacterSheetSlot/index.tsx:22]: Question: Will this be stored in context for the actual implementation?

File: src/data-accessor.ts [hunk 3]
```diff
@@ -41,6 +42,11 @@ export const statsForLevel = (level: CharacterLevel): LevelStats => {
   return stat;
 };
 
+// TODO: This is really silly, this should be in a map
```
Comment (AndaxH) [inline src/data-accessor.ts:45]: 😂

**Review by AndaxH — APPROVED** [review]
Seems good

File: src/components/CarouselClassSpells/index.tsx (hunk 1 above)
Comment (bqx) [inline src/components/CarouselClassSpells/index.tsx]:
```suggestion
export default function EquipClassSpells({ onClose, onSelectItem }: Props) {
```

**Pull request merged**